config.toml

```diff
@@ -10,6 +10,8 @@ theme = "hyde"
 [Menus]
   main = [
     {Name = "Contributing", URL = "/contribute/"},
+    {Name = "Github", URL = "https://github.com/sre-paris/reliability.re"},
+    {Name = "Twitter", URL = "https://twitter.com/ReliabilityR"}
   ]
```

Comment:
By the way, if you could also include the link to Telegram https://t.me/ReliabilityReport  would be awesome!